### DISCUSSION TITLE
行人最大？！

### DISCUSSION THREAD

**Discussion by WeiYen1103** [1101-GS4719-C/Python-Community-GS4719-C#1762] [General]
6月的時候頒布了，未禮讓行人罰6000的法令，起初是為了解決，台灣的交通環境遭人詬病多年，先前更被外媒評為「行人地獄」的這個問題，提高行人的安全保護，我覺得這項法令是好的，在路上很明顯的車子不會為了趕紅綠燈而盲目的直接穿越人行道，而會先注意有沒有人要經過，但近期有一些人仗著「行人最大」的說詞霸占路權，直接擋在路中央，影響了交通，我認為很多事情是一體兩面的，儘管這項法令本身是為了行人安全而設置，但一定會有人將其視為「行人最大，你不能動我」，
但我昨天看到新聞，若是有這種情況發生，行人站在馬路中間阻擋車輛通行，是會構成強制罪的，希望大家能將心比心，不要一時貪快就沒注意到行人，也不要仗著現在有法令就刻意造成交通的混亂。

**Comment by Chengbosco** [comment]
本來就是車子想只貪快，完全不管行人，所以才那麼多車禍，而且台灣交通規劃本來就有問題了，突然加這條法律確實太昌促了，雖然想要扭轉行人地獄的形象但確實不妥，之前地科院週會請來的人也是，就放一堆汽車撞爛行人和摩托車的影片，阿也不講講為什麼會這樣，總覺得我想要安全走在路上還要很久。
是說我以前在台北只是正常過紅綠燈就被按喇叭真的是凸 : ) 凸

**Comment by Sherry9106** [comment]
最近有一個新聞是在說一個國中生參加全國美術比賽拿第一名，他畫的主題是帝王條款，一個穿著像皇帝的人過馬路，所有人和車子都停下，但他的核心理念是希望行人和車子互相尊重，而不是行人最大。像我自己過馬路的時候都會注意車輛，如果有車在等我也會趕快過，不讓他們等太久，人真的要互相。附上新聞網只給大家:https://news.ltn.com.tw/news/life/breakingnews/4518984

**Comment by 111426027** [comment]
我覺得這條法律的頒布是好的，但前提是行人過馬路時不要慢慢走!如果讓行人先走還慢慢走的話，真的會超生氣。像汽車任我先走的時候我都會加快腳步，希望大家能相互體諒。

**Comment by Patsy17** [comment]
台灣的行人議題越來越受到重視，不過都是在發生許多起行人走斑馬線還是被撞到發生重大傷亡後才開始修法，
個人認為不論是行人還是汽機車都應該要互相尊重，汽機車禮讓行人之餘，行人在自己行動能力內應該也要盡快穿越馬路～

**Comment by Yanxxx7979** [comment]
我個人認為，這個政策出發點雖然是好的，但完全治標不治本，行人地獄之所以存在，根本上的原因其一是，在考取駕照時，駕駛並未受到正確且嚴謹的道路意識，例如互相禮讓等等
而另外一個我覺得更加嚴重的是道路規劃問題，在一開始便沒有將行人納入考量，更多的是站在大車角度進行規劃，而造成行人沒有路權
而現如今，使用罰款要求汽車禮穰，雖然能抵擋一時，但無不可能是在製造行人與大車的對立，更沒有解決真正的問題

**Comment by mrweixd** [comment]
這項法令初衷是為了提高行人安全，但近期有些人以「行人最大」為由，濫用路權，擋在路中央影響交通。雖然法令有助於改善交通環境，但應該注意不要讓其被誤用，並呼籲大家相互尊重，不要因為法令而造成交通混亂。互相理解、守規矩，才能實現更安全有序的交通環境。

**Comment by davidteng00000** [comment]
我認為對不禮讓行人的汽機車做罰鍰完全無法解決台灣的行人地獄問題，最根本的問台還是出在台灣人的駕駛觀念與交通的規劃，例如政府應該多設立人行道並重新審視一些事故頻發地區的道路設計

**Comment by Ryanlolry** [comment]
我認為雖然已訂立相關法規規定汽車必須讓行人通過，但據我觀察在一些不大的路口，如校內道路，真正做到的車輛屈指可數，為了保障行人的安全而去設立這項法規是好的，但要嚴格執行在每個地方，並且配合改善交通設計，來減少人車爭道的窘境。

**Comment by vchien03** [comment]
其實這個法令頒布之後大概一、兩個月內是最有感的，後來還是常常看到斑馬線前有四、五個人在等，但車輛仍然快速開過。台灣用路人沒有正確的觀念，就難以達到行人與車輛互相禮讓。

**Comment by yuchien-lin** [comment]
我覺得這條法令是好的，但是在某些地方不太合適，假設有陸陸續續的路人穿過馬路,，那就會造成車子無法行動，然後後方的車也無法往前就直接塞住了，所以我覺得在某系路段可能採用更改紅綠燈的方法比較好，像是桃園區的市區就有一個十字路口，採用更改紅綠燈的方法，讓行人能過汽車也能過。

**Comment by B1nn9** [comment]
題外話，在這條法令頒布然後搞到社會沸沸揚揚之前，我以前有觀察到其實在台北市普遍汽機車都會禮讓行人，大家走在路上其實都還挺放心的。但不知為何這現象一出了台北後彷彿就消失，在其他地區過馬路的時候就得自己多注意..我在想會不會是因為不同地區考駕照時，對於行人安全這塊所做的宣導教育程度也有所不同？

**Comment by alvin119** [comment]
臺灣被稱為行人地獄跟駕駛的素質有關，
我覺得臺灣的駕照考取應該改成跟日本一樣，
除了有更明確的駕照分級還要定期實施更換駕照的檢定

**Comment by chuanmin0317** [comment]
在法令頒布後，我自己是有感覺到在過馬路的時候有變的安全很多，畢竟之前有些機車可能會在我過馬路時為了搶快從我前面穿過，不過法令頒布後，規定要從行人後方穿過，此舉我覺得大幅減低過馬路時要一直走走停停的問題

**Comment by YANRONG0123** [comment]
我覺得這樣的條款只是加強駕駛認知何謂是錯誤的駕駛行為，但應該要從考取駕照的那個過程加以審核才對，目前臺灣的交通法規還是非常不完善，很多人就算犯下錯誤，還是有可能理直氣壯的覺得自己沒有問題。

**Comment by YifenLin** [comment]
自從這條法律頒布之後，我自己騎車右轉要過斑馬線都會特別注意要暫停或減速，不過可能因為大家還沒習慣，我有看過右轉暫停等待行人通過反而被後面也要右轉的車子追撞而發生車禍，提醒大家開車要注意路況小心安全歐。

**Comment by 110101501** [comment]
有這項的法案的頒布是好事，能讓大眾更意識到行人路權的重要性。
這項法案剛發布沒多久後，有一次我在士林石牌一帶開車要轉彎。由於馬路過長，沒注意到在遠處正有行人起步。雖然行人距離我很遠，但我還是因此被開了未禮讓行人的罰單，被開罰了3600⋯

**Comment by LinChloe** [comment]
我覺得政府在說行人最大前，是不是應該先把人行道修好。
台灣許多道路沒有人行道，或著人行道不連貫、停滿機車、有電線桿擋著等問題。
想要行人走路安全麻煩先把人行道修好修寬一點。
至少要讓嬰兒車和輪椅能兩台並行的寬度。
不然行人都走到馬路上，行人想走路安全簡直是癡人說夢話。

**Comment by yxin214** [comment]
我支持這個法令，畢竟若汽車與行人相撞，行人絕對會受到比較大的傷害。但若行人仗著這項法規隨意穿越馬路或是忽略號誌等，即扭曲了這項法令的美意。行人與駕駛都是用路人，應彼此尊重、互相禮讓。

**Comment by thomasxshen** [comment]
台灣的行人霸王條款旨在保護行人的安全，如行人於斑馬線上具有優先通行權。這些條款在提高行人安全和降低交通事故方面扮演了重要角色，不過，它們也可能導致一些司機感到不滿，尤其是在交通繁忙時，重要的是要衡量其對於提升交通安全的效果以及是否合理地平衡了行人與駕駛者的權益。合理的設計與執行應能有效降低交通事故發生率，並提升整體的道路使用體驗。這需要政府進行持續的評估和必要時做出調整，以確保法規能夠適應不斷變化的交通環境和社會需求。

**Comment by wanka2030** [comment]
不得不說行人條款只是在保護行人的安全，並沒有指定落實行人的教育安全，與其讓原本不守法的行為強制糾正，不如從頭到尾改善教育素養讓行人能有明確的安全意識，不然同樣的事情只會一而三的重演。況且行人的安全不能只是受到法律的眷養兒肆無忌憚，行車的人也需要保有明確的安全意識，來讓交通維持安全、善良的環境。

**Comment by hbh10601** [comment]
基本上我覺得很大部分是教育問題，這很像是一種根深蒂固的文化影響，看看日本、美國和一些歐洲國家，大家都是你讓我、我讓你，根本因為這樣引起軒然大波，但台灣一頒布這條法律，我聽過很多長輩第一個反應都是：「那這樣走路的就會開始囂張啊！開車的人一定吃虧啊！」、「白癡法律！白癡政府！」，根本就沒有好好理解法條背後的意義，我很想問那群人，你們就一輩子待在車上都不會走路嗎？
因此我認為，台灣人在同理心的部分還需要加強。

**Comment by happy246810** [comment]
雖然我支持這項法律的成立，畢竟行人與車輛在發生車禍時，行人絕對會受到較大的傷害。同時，我也覺得禮讓行人這個觀念應該是要所有人都要知道的，頒布法律只是一個過渡期。但我認為行人也不應該仗著有法律保障，就如此的囂張跋扈，因此我認為駕駛跟行人都應該有互相禮讓的習慣。

**Comment by Eileen050824** [comment]
我認為這個法律的頒布本意沒什麼問題！行人雖是容易受到較大生理傷害的一方，卻也不應該把這法律當胡作非為的靠山，同時提醒行人不能太超過，應在具一定風險的環境注意行為並為結果負責。只要駕駛與行人雙方彼此體諒，相信可以減少很多不必要事故與糾紛。

**Comment by Hahahahbla** [comment]
新的行人禮讓法令是為了提高交通安全，但有人濫用「行人最大」的觀念擋車，影響交通。雙方應互相尊重，遵守法規，以和平共享道路。行人應注意交通狀況，車輛應禮讓行人。尊重彼此，共同創造安全的交通環境。

**Comment by YuanWang08** [comment]
會有這條法律也是體現出了台灣當今的社會情形，就是不讓行人的車遠大於不讓車的行人。其實這種現象我覺得是長期下來的社會文化產生的，不只是會有車闖紅單搶快，更多的應該是行人為方便隨意穿越馬路，只是因為不罰，出事了也不會是行人對撞，一定是撞到什麼車，然後新聞就爆出來，沒事不會有人說，但出事一定有車，才會造成現在這種情況。

**Comment by yyujimmy** [comment]
這項法令的實施確實是為了提高行人的安全保護，並改善台灣交通環境。然而，像您提到的一樣，任何法令都可能被某些人利用或誤用，甚至導致不當行為的發生。

對於行人來說，應該尊重交通規則和他人的權利，不應該因為法令的存在而肆意地阻擋車輛通行或影響交通流暢。同時，對於駕駛者來說，也應該注意行人的安全，遵守交通規則，給予行人足夠的通行空間和尊重。

在交通安全方面，雙方都有責任，需要共同努力才能創造更安全和更和諧的交通環境。希望人們能夠互相理解，共同遵守交通規則，共同維護交通秩序，從而達到保護行人安全和改善交通環境的目標。

**Comment by willychiuchiu** [comment]
這項法令的實施確實是為了提高行人安全，但如你所說，任何法令都可能被人以不同方式理解和濫用。重要的是教育和培養大眾對於道路安全的意識，以及相互尊重彼此的權益。盲目地執行法令或者濫用法律規定都不是解決問題的最佳方式，我們需要更多的溝通和理解，以創造更和諧、安全的交通環境。

**Comment by KH1126** [comment]
這項法令原本是為了提高行人的安全保護，解決台灣交通環境長期以來被詬病的問題。
然而，一些人可能會誤解這項法令，將其視為可以隨意佔用路權的理由，從而對交通造成困擾。
因此我認為也需要有對應的措施。

**Comment by jim1743** [comment]
我認為這項法令的實施立意真的不錯，畢竟台灣道路事故的問題真的一再發生，但要立馬改變台灣道路環境是不可能的，畢竟改善交通這種事，必須依靠駕駛與行人的道路交通觀念正確才行，因此我認為未來政府要多多宣導正確的道路駕駛觀念給名眾知道才行。

**Comment by thomasxshen** [comment]
我自己覺得這只是政客用來搪塞民眾的垃圾法條，因為他們根本就不想去改善交通，所以只好用一堆不合理的嚴刑峻法來約束用路人，好笑的是這些法條還常自相矛盾，比如行人霸王條款，其實不論你應注意或未注意，能不能反應，撞到行人就很容易被判罰，我自己覺得很荒謬，因為這完全就是給法官自由心證的東西，而台灣的法官又很偏向行人的權益，導致很多無辜的用路人因此被罰。

**Comment by Mitochondrionjessica** [comment]
法律是道德的最低標準，但不代表有法律保障的行人可以為所欲為。

**Comment by YOGA012** [comment]
我非常支持新的未禮讓行人罰6000的法令，這有助於提升台灣的交通環境和行人安全，解決長期以來被詬病的「行人地獄」問題。近來，車輛在斑馬線前禮讓行人的情況確實增加了，這是一個很好的進步。然而，也有少數人藉著「行人最大」的說法霸占路權，影響交通流暢，這種行為並不應該鼓勵。法律的設立本意是保障大家的安全，因此希望行人和駕駛者都能互相理解和尊重，不要因為一時的便利而忽視他人的安全和权利。只有大家共同努力，台灣的交通環境才能真正變得更好。

**Comment by alexender0104** [comment]
我認為儘管這項法令有助於提高行人的安全意識，但在執行過程中，確實可能會出現一些人利用法令中的某些條款來影響交通，這可能導致不必要的混亂和危險。重要的是，無論是行人還是駕駛者，都應該互相尊重和理解對方的立場。行人應該遵守交通規則，不要過度利用自己的權利，以至於影響交通流暢。駕駛者則應該注意行人的存在，給予他們足夠的空間和尊重。

**Comment by Shang621** [comment]
在道路交通中，行人最大，因為他們是最脆弱的道路使用者之一。行人包括步行者、跑步者、推車、行動不便的人等，他們沒有防護裝備，無法與車輛相比。因此，駕駛人應該給予行人最大的尊重和優先權，並遵守交通規則，特別是在人行道和過街道路上。此外，行人也應該注意交通，穿著明亮顏色、使用行人穿越處等方式提高自身可見性，確保安全。行人最大的原則提醒我們，在道路上相互尊重和體貼，共同創造安全的行車環境。

**Comment by yubo0oo** [comment]
這項法令的實施確實是為了提升行人安全，並改善台灣的交通環境。然而，一些人可能會誤解這項法令，認為他們有絕對的權利在馬路上行走而不需考慮其他交通參與者的權益。這樣的行為不僅影響了交通流暢，也可能導致危險情況的發生。

正確的行為應該是相互尊重和遵守交通法規，行人應該注意不要阻礙交通流動，車輛司機也應該給予行人應有的安全空間和優先權。這需要大家共同努力，以保障交通安全和便利。

**Comment by 710388** [comment]
確實該這麼作，畢竟直觀地看在馬路上確實行人是相對弱勢，且較容易受傷

**Comment by poiu6fi** [comment]
之前新聞好像有報導說有些人會因此站在斑馬線拍網美照，汽車駕駛又因為這項法規無法前行，導致交通阻塞。這個法案的出發點固然是好的，不過如何從濫用權力跟保護行人兩者之間取得平衡，我覺得值得大家去思考

**Comment by OHoln** [comment]
雖然文中提到的行為的確不恰當，但我也認為目前台灣對於行人的保護與規範還有很大的改進空間，更重要的是必須改變多數駕駛的習慣，才能幫行人創造一個更好的環境

**Comment by YeeDaniel** [comment]
「行人最大」是一種提醒司機和其他交通參與者注意行人安全的標語。這個概念強調在道路使用中，行人應該被優先考慮，尤其是在過馬路和穿越交叉路口時。這樣的標語旨在提高人們對行人權利的認識，減少交通事故，營造更安全的道路環境。通過遵守這一原則，我們可以共同創造一個更友好的交通氛圍。
